doc/releases/changelog-dev.md

```diff
@@ -16,6 +16,9 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* `CircuitGraph` can now handle circuits with the same operation instance occuring multiple times.
```

Comment:
Might be worth adding another entry under "Breaking Changes".

Reply:
added.